### ISSUE TITLE
Email comes in wrong encoding if the interface language russian

### ISSUE THREAD

**Issue by id777** [phpservermon/phpservermon#254]
Hello! The PHP Server Monitor installed a Russian interface.
Here are some letters come to the mailbox gmail: https://hsto.org/files/457/3df/d58/4573dfd5851f4f3086057e729d9c904a.png (screenshot).

Incorrect coding?

**Comment by sadortun** [comment]
Hi @id777 can you try to investigate more? Can you provide the current expected, and actual encoding ?

**Comment by webbymaster** [comment]
I set the Russian language in the settings, but the E-Mail messages come in English. I tried to translate in the file en_US.lang.php, I still get it in English. Editing a file /src/includes/functions.inc.php this leads to the fact that the emails do not arrive at all. It feels like the file ru_RU.lang.php it is not picked up by the system until the end. The translation only works for a part of the script. Help me, tell me where to make the translation, so that all the letters are in Russian/ Thanks!
